README.md

```diff
@@ -92,7 +95,7 @@ curl -fsSL https://d2lang.com/install.sh | sh -s -- --uninstall
 Alternatively, you can install from source:
 
 ```sh
-go install oss.terrastruct.com/d2
```

Comment:
by default, is this not equivalent to the changed? i.e. does it not get latest by default?

Reply:
The purpose of `@latest` is to force ignoring the current go module. See docs at https://go.dev/doc/go-get-install-deprecation

The other change is using the import path of the d2 executible and not the library.